### PR TITLE
Update webhost to 1.0.13146

### PR DESCRIPTION
**Pull request by yojagad**
Preparing core tools for https://github.com/Azure/azure-functions-host/releases/tag/v1.0.13146

### CODE REVIEW

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 1]
```diff
@@ -281,8 +281,8 @@
       <HintPath>..\..\packages\Microsoft.Azure.NotificationHubs.1.0.7\lib\net45-full\Microsoft.Azure.NotificationHubs.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.WebJobs, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Core.2.4.0-beta1-11673\lib\net45\Microsoft.Azure.WebJobs.dll</HintPath>
+	<Reference Include="Microsoft.Azure.WebJobs, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (ankitkumarr) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:284]: minor spacing issue

**Comment by pragnagopa** [comment]
this is still not merged? blocked on anything?

**Comment by yojagad** [comment]
@pragnagopa last I remember, there were signing issues and it was handed off to the next release owner.

**Comment by pragnagopa** [comment]
Ok. Thanks for the update. I will follow up tomorrow.

**Comment by ankitkumarr** [comment]
This (same commit) was merged through https://github.com/Azure/azure-functions-core-tools/pull/2211.

**Pull request closed without merging**